README.md

```diff
@@ -14,8 +14,8 @@ An Automatic Page Generation Tool http://jxnblk.com/doctor-mark
   - Meta description
   - Meta author
   - Meta keywords
-  - Github link
-  - NPM link
+  - GitHub link
+  - npm link
```

Comment:
Fair enough, I usually go by https://docs.npmjs.com/misc/faq#is-it-npm-or-npm-or-npm and always lowercase it everywhere since it's their business name.